README.md

```diff
@@ -33,8 +33,10 @@ a tchannel bridge, and wrap there encoded handler function with `tracedHandler`
 
     serverThriftChannel.register(server, 'Echo::echo', context, bridge.tracedHandler(
         (context, req, head, body, callback) => {
-            // context will contain an 'openTracingSpan' field that stores the tracing span for the inbound request.
-            // Your handler code goes here.
+            // context must be an object with the methods 'setSpan', and 'getSpan' which are responsible,
```

Comment:
"context must implement 'setSpan' and 'getSpan' which are responsible for setting and getting the span on the context respectively."

Reads easier.